src/modules/test/README.md

```diff
@@ -8,4 +8,7 @@ Although `modulestest` does not require any runtime dependencies, generating the
 1. `modulestest` or `modulestest testplate.json` (equivalent) tests all modules which have [test recipe](https://github.com/Azure/azure-osconfig/tree/main/src/modules/test/recipes) and [MIM](https://github.com/Azure/azure-osconfig/tree/main/src/modules/mim) defined.
 2. `moduletest <module name>` Example: `modulestest tpm` will perform the [test recipe](https://github.com/Azure/azure-osconfig/tree/main/src/modules/test/recipes) defined for the _tpm_ module.
 3. `moduletest <module.so>` Example: `modulestest /azure-osconfig/build/modules/tpm/src/so/tpm.so` performs a basic test on the module ONLY, this ensures the MMI interface is properly defined. No payloads are sent to modules, we simply load the module, ensure _MMIGetInfo_ contains the correct module definitions.
-4. `moduletest <module.so> <mim.json> <recipe.json>` Example: `modulestest "/azure-osconfig/build/modules/tpm/src/so/tpm.so" "/azure-osconfig/src/modules/mim/tpm.json" "/azure-osconfig/src/modules/test/recipes/TpmTests.json"` Equivalent to #2
\ No newline at end of file
+4. `moduletest <module.so> <mim.json> <recipe.json>` Example: `modulestest "/azure-osconfig/build/modules/tpm/src/so/tpm.so" "/azure-osconfig/src/modules/mim/tpm.json" "/azure-osconfig/src/modules/test/recipes/TpmTests.json"` Equivalent to #2
+
+# Known limitations
+  - Every test definition in a recipe is executed in its own session. Tests will not function as expected if relying on subsequent definitions sharing the same session.
```

Comment:
>Every test definition in a recipe is executed in its own sessio [](http://example.com/codeflow?start=4&length=63)

There is no such thing as "test definition". Those are MIM object payloads.

Suggest to change to this:

"Each object payload in a Test Recipe is currently executed in its own module session (MmiGetInfo and MmiOpen are executed before this payload and MmiClose after it). This is a temporary limitation that will be removed in the future (so that all objects payloads in a recipe will be executed into one module session)."